apps/demo-game/src/pages/admin/games/[id].tsx

```diff
@@ -429,8 +430,8 @@ function ManageGame() {
 
       <div className="mt-4 max-w-sm">
         <div className="font-bold">Players</div>
-        <div className="mt-2 flex flex-col gap-2">
-          {data.game.players.map((player, ix) => (
+        <div className="mt-2 flex flex-col gap-4">
+          {game.players.map((player, ix) => (
             <div key={player.id} data-cy={`player-${ix}`}>
               <PlayerCompact key={player.id} player={player as Player} />
```

Comment:
only the outermost item within a loop needs to have a key={} property, unless you are using key as a prop, which you should never do as it is a react reserved thing

```suggestion
              <PlayerCompact player={player as Player} />
```

Reply:
Ah I see, I did not know that thanks a lot!